data/certChooser.js

```diff
@@ -25,6 +25,10 @@ for (var nickname in data.certs) {
 // assume they wanted to cancel.
 data.cancelled = true;
 
+function selectPassword() {
+  setTimeout(function(){ document.getElementById("certPassword").focus(); }, 100);
```

Comment:
It is needed at least when dialog is opened. On certificate change direct call of focus() works.